Reorg NFT page

filippoweb3 opened the pull request.

Commit:
Removed expect chaos statement

Commit:
Minor edits

Commit:
Minor edit

DrW3RK commented:
Are you still working on this? If so, I can review it again after it is finished. Please add the "in-progress" label

filippoweb3 commented:
Yeah, I was thinking of changing more stuff, but I think I will leave it for now. I removed "expect chaos" in the middle of a paragraph, and added some tiny edits.
